Cartfile

```diff
@@ -5,3 +5,4 @@ github "kif-framework/KIF"
 github "st3fan/SWXMLHash"
 github "rs/SDWebImage"
 github "norio-nomura/Base32"
+github "DaveWoodCom/XCGLogger" ~> 1.8
```

Comment:
I think for master it is fine to not specify any pinned version and just grab the most recent one. If things break because libraries get updates, we can deal with it. Either by updating our code or by pinning to a prevous version if that is more appropriate.

But this raises an intersting issue: I think every time we do a distribution build (aurora or release) we should actually pin the versions in the `Cartfile`. So that we can have reproducible builds that do not need any guesswork. (With what version of SDWebImage was this compiled when we released this 3 weeks ago!?)